packages/stark-ui/src/modules/collapsible/components/collapsible.component.ts

```diff
@@ -0,0 +1,95 @@
+import { Component, EventEmitter, HostBinding, Inject, Input, OnInit, Output } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+import { StarkCollapsibleOptions } from "@nationalbankbelgium/stark-ui";
+
+/**
+ * Name of the component
+ */
+const componentName: string = "stark-collapsible";
+
```

Comment:
Can you add a JSDoc line to explain in few words what the component is about :-)